datahub-web-react/src/graphql/glossaryTerm.graphql

```diff
@@ -28,6 +28,19 @@ query getGlossaryTerm($urn: String!, $start: Int, $count: Int) {
                 }
             }
         }
+        childTerms: relationships(input: { types: ["IsA"], direction: INCOMING, start: $start, count: $count }) {
```

Comment:
Minor: This field may be confusing because there are technically both "IsA" and "HasA" children terms from this one. 

Would it be clearer to say

isAChildren? 

or 

hasAChildren? 

Just curious to hear your thoughts. We are also okay to do this as a followup

Reply:
yeah that makes sense - I think `isAChildren` for this case is pretty understandable and clear - will make a quick change and push up